src/structure/structure.ts

```diff
@@ -0,0 +1,22 @@
+import { apiClient } from '../erudio';
+
+export interface Options {
```

Comment:
@JERILJOSE96 every field in `Option` interface should be optional, there's no need of using `filter` when you want to specify `page` (and vice versa)